src/css/reset.scss

```diff
@@ -0,0 +1,117 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
gonna chuck this whole file since I never really figured out hoe to use it in the way I intended.